pontos/version/commands/_cargo.py

```diff
@@ -68,9 +76,12 @@ def update_version(
 
     def get_current_version(self) -> Version:
         (_, document) = next(self.__as_project_document(self.project_file_path))
-        current_version = self.versioning_scheme.parse_version(
-            document["package"]["version"]  # type: ignore[index, arg-type]
-        )
+        try:
+            version = document["package"]["version"]  # type: ignore[index, arg-type]
+        except NonExistentKey:
```

Comment:
You could just use a KeyError from the stdlib here (https://github.com/python-poetry/tomlkit/blob/master/tomlkit/exceptions.py#L180). Would make it independent of tomlkit and would work with all dicts too.
```suggestion
        except KeyError:
```

Reply:
I was not aware, that `NonExistentKey` is also inherited from `KeyError`!